### DISCUSSION TITLE
Issue with WRITE_APPEND not declared in scope

### DISCUSSION THREAD

**Discussion by jedhatchell** [pschatzmann/TinyFTPClient#4] [Q&A]
I'm using your library with an ESP8266.  I'm very new to this so I appreciate your help and patience.  I've had very good luck using this write data to my ftp server.  This works great.  I'm now trying to use the append mode to add data to a file already created.  I can't get past the compilation, I get a "WRITE_APPEND was not declared in this scope" error.  Below is a small code sample which has the issue.  If I change WRITE_APPEND to WRITE_MODE, I don't get this error.  I appreciate any help.

#include <ArduinoFTPClient.h>
#include <ESP8266WiFi.h>
WiFiClient cmd;
WiFiClient data;
FTPClient client(cmd, data);

void setup() {
     client.begin(IPAddress(192,168,1,10), "user", "password");
    FTPFile file = client.open("/test.txt", WRITE_APPEND);
    
    file.close();
    client.end();

}

void loop() {
 
}

**Comment by pschatzmann** [comment]
I confirm that WRITE_APPEND_MODE is the correct value.
You can also double check this is the [class documentation](https://pschatzmann.github.io/TinyFTPClient/docs/html/class_f_t_p_client.html#ab881602d3b097bbc96f43cc7e5f08ec8)

**Reply by jedhatchell** [reply]
Thanks for your quick response.  It does work now.  The confusion on my part comes from the code snippet you show in the main page of the repository.  It shows "WRITE_APPEND", not "WRITE_APPEND_MODE".  

Thanks again.

    WiFiClient cmd;
    WiFiClient data;
    FTPClient client(cmd, data);

    client.begin(IPAddress(192,168,1,10), "user", "password");
    FTPFile file = client.open("/test.txt", WRITE_APPEND);
    char buffer[100];
    for (int j=0;j<100;j++){
        sprintf(buffer, "test %d", j);
        file.println(buffer);
    }
    file.close();
    client.end();

**Reply by pschatzmann** [reply]
I have updated the documentation: Thanks for letting me know...
